src/Umbraco.Web.Common/Builder/UmbracoBuilderExtensions.cs

```diff
@@ -30,6 +30,9 @@ public static IUmbracoBuilder WithConfiguration(this IUmbracoBuilder builder)
         public static IUmbracoBuilder WithCore(this IUmbracoBuilder builder)
             => builder.AddWith(nameof(WithCore), () => builder.Services.AddUmbracoCore(builder.WebHostEnvironment, builder.Config));
 
+        public static IUmbracoBuilder WithHostedServices(this IUmbracoBuilder builder)
+            => builder.AddWith(nameof(WithHostedServices), () => builder.Services.AddUmbracoHostedServices());
+
```

Comment:
`AddWith` is such an awkwardly named method, the individual methods using the `With` prefix also . `Set` is far more succinct.

Reply:
Suggest this is amended or otherwise outside of this PR, and it'll affect more than just what's being included here.

Reply:
Yeah, agree, was just highlighting.